### DISCUSSION TITLE
Cloudflare Pages and sanitize-html

### DISCUSSION THREAD

**Discussion by joaoguidev** [remix-run/remix#8660] [Q&A] [answered]
Hello.

I'm having problems trying to run sanitize-html on on my Cloudflare Pages. 

sanitize-html needs to have access to node::process and Cloudflare pages does not provide that.

The only way I manages to do this work was by creating a file **sanitize.server.jsx** and globaThis the process. Is it **SAFE** to do this?

![image](https://github.com/remix-run/remix/assets/63625334/667894f4-d96d-4dd4-9932-02fd5d078bb2)

Later i import this function to use on my loaders and actions.

If I try to use sanitize-html on Cloudflare Pages without doing this i get the error: `service core:user:worker: Uncaught ReferenceError: process is not defined`

**Comment by sergiodxa** [comment, marked as answer]
You either mock `globalThis.process` as you did or you need to find another library that doesn't depend on a Node.js built-in feature like `process`.

**Reply by joaoguidev** [reply]
Hello Sergio. Thank you for replying.

I was trying to use Isomorphic-dompurify but this is not working with Remix for now. I opened on issue regarding this [here](https://github.com/remix-run/remix/issues/8628)

I'm going to try other ones. If you have any suggestions it will be welcomed.

Thank you
